src/jetstream/pull_subscription.rs

```diff
@@ -161,6 +161,16 @@ impl PullSubscription {
     pub fn iter(&self) -> Iter<'_> {
         Iter { subscription: self }
     }
+
+    fn process_no_messages(&self, message: Option<Message>) -> Option<Message> {
```

Comment:
You can also get request timeout messages that you would need to handle as well.

Reply:
Thanks! Good catch. Will add them.

Reply:
Fixed.